.gitmodules

```diff
@@ -0,0 +1 @@
+
```

Comment:
Do we need this file if its empty?

Reply:
Probably not.  I tried once to use the submodule so it was generated but I decided not use it.  I'll take it out.  Thanks!